python/pyweld/setup.py

```diff
@@ -30,6 +30,8 @@ def move_file(self, filename, directory):
         source = filename
         dir, name = os.path.split(source)        
         destination = os.path.join(self.build_lib + "/" + directory + "/", name)
+        makedir = self.build_lib + "/" + directory
+        os.system("mkdir -p {}".format(makedir))
```

Comment:
The build directory should be created automatically on setup.py install.
I'm guessing you're doing this because setup.py develop is failing?
The PR #240 fixes setup.py develop.

Reply:
Ah okay, I'll revert this then.